src/platform/packages/shared/shared-ux/avatar/solution/src/solution_avatar.tsx

```diff
@@ -56,16 +63,27 @@ export const KibanaSolutionAvatar = (props: KibanaSolutionAvatarProps) => {
     icon.iconType = `logo${props.name.replace(/\s+/g, '')}`;
   }
 
+  const { euiTheme } = useEuiTheme();
+  const styles = {
+    base: css(useEuiShadow('s')),
+    xxl: css`
+      ${useEuiShadow('m')};
+      line-height: 100px;
+      width: 100px;
+      height: 100px;
+      border-radius: 100px;
```

Comment:
Can you use EUI size variables to calculate this values?

Reply:
I could do `xxxl * 2`, but that is only `48 * 2`. LMK if you would still prefer that.